### PR TITLE
DOCS-2826: Add updates to CLI page from QAing

### PR DESCRIPTION
**Pull request by sguequierre**

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -1016,7 +1019,7 @@ The `organizations` command allows you to list the organizations your authentica
 
 ```sh {class="command-line" data-prompt="$"}
 viam organizations list
-viam organizations api-key create --org-id <org-id> [--name <key-name>]
+viam organizations api-key create --org-id=<org-id> [--name <key-name>]
```
Comment (npentrel) [inline docs/cli.md:1022]:
```suggestion
viam organizations api-key create --org-id=<org-id> [--name=<key-name>]
```

File: docs/cli.md [hunk 2]
```diff
@@ -1071,7 +1074,7 @@ viam packages export --org-id=<org-id> --name=<package-name> --version=latest --
 | -------- | ----------- | ------------------- | -------- |
 | `--org-id` | The organization ID of the package | `upload`, `export` | **Required** |
 | `--name` | The name of the package | `upload`, `export` | **Required** |
-| `--version` | The version of the package or `latest` | `upload`, `export` | **Required** |
+| `--version` | The version of the package | `upload`, `export` | **Required** |
```
Comment (npentrel) [inline docs/cli.md:1077]: I think training scripts aren't versioned and I don't actually know where versions for ML models etc are currently displayed. So I think having that is quite important.
```suggestion
| `--version` | The version of the package or `latest` | `upload`, `export` | **Required** |
```

File: docs/cli.md [hunk 3]
```diff
@@ -1051,9 +1054,9 @@ See [create an organization API key](#create-an-organization-api-key) for more i
 The `packages` command allows you to upload packages to the Viam cloud or export packages from the Viam cloud.
 
 ```sh {class="command-line" data-prompt="$"}
-viam packages upload --org-id=<org-id> --name=<package-name> --version=latest --type=ml_model --path=.
+viam packages upload --org-id=<org-id> --name=<package-name> --version=1.0.0 --type=ml_model --path=.
 
-viam packages export --org-id=<org-id> --name=<package-name> --version=latest --type=ml_model --destination=.
+viam packages export --org-id=<org-id> --name=<package-name> --version=1.0.0 --type=ml_model --destination=.
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/cli.md:1059]: I realized I just tested this with some dummy values on the train to make sure the flags were correct but trying now with viam-dev I'm getting server errors even if I don't use lastest (for example `viam packages export --org-id=e76d1b3b-0468-4efd-bb7f-fb1d2b352fcb --name=hand-model --version=2024-07-16T12-16-38 --type=ml_model --destination=./test-ml-export`) so I would say no it doesn't work. Thanks for picking up on that

[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/cli.md:1059]: Should I report to ml team?

File: docs/cli.md (hunk 2 above)
Comment (sguequierre) [reply docs/cli.md:1077]: Ok interesting. Whatever I was doing on the train I concluded that latest wasn't working/accepted but I must have been confused

File: docs/cli.md [hunk 4]
```diff
@@ -1302,7 +1330,7 @@ Info: Successfully registered auth application
   "secret": "supersupersecretsecret"
 }
 
-viam auth-app update --org-id=z1234567-1a23-45a6-a11b-abcdefg1234 --application-id=1234a1z9-ab2c-1234-5678-bcd12345678a --redirect-uris="https://test.com","https://test2.com"
+viam auth-app update --org-id=z1234567-1a23-45a6-a11b-abcdefg1234 --application-name="julias app" --application-id=1234a1z9-ab2c-1234-5678-bcd12345678a --redirect-uris="https://test.com","https://test2.com"
```
Comment (npentrel) [inline docs/cli.md:1333]: So auth-app also has a `get` command now. Did you check all the commands on what sub commands they now support?
```
➜  viam-docs git:(DOCS-2789/capture-sync-image-data) ✗ viam auth-app
NAME:
   viam auth-app - manage third party auth applications

USAGE:
   viam auth-app command [command options] [arguments...]

COMMANDS:
   register  register a third party auth application
   update    update a third party auth application
   get       get configuration for a third party auth application
   help, h   Shows a list of commands or help for one command
```

Comment (npentrel) [reply docs/cli.md:1333]: `viam train` also has a bunch of new command options. Can you please go through them and add the missing ones?

File: docs/cli.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/cli.md:1059]: Yes. Please file a ticket and cc me on it

File: docs/cli.md [hunk 5]
```diff
@@ -1101,20 +1104,23 @@ viam machines part restart --machine=<machine id> --part=<part id>
...
+--organization "Robot's Org" --location myLoc
+
+# stream logs from a machine part
+viam machines part logs --machine 82c608a-1be9-46a5-968d-bad3a8a6daa \
+--organization "Robot's Org" --location myLoc --part myrover-main --tail true
 
 # stream classifications from a machine part every 500 milliseconds from the Viam Vision Service with classifier "stuff_detector"
 viam machines part run --machine 82c608a-1be9-46a5-968d-bad3a8a6daa \
---organization "Robot's Org" --location myLoc --part "myrover-main" --stream 500ms \
+--organization "Robot's Org" --location myLoc --part myrover-main --stream 500ms \
```
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/cli.md:1123]: Youve added the equal signs in most other places, so I think if we do that, we should be consistent with it

File: docs/cli.md [hunk 6]
```diff
@@ -1101,20 +1104,23 @@ viam machines part restart --machine=<machine id> --part=<part id>
...
 viam machines status  --machine 82c608a-1be9-46a5-968d-bad3a8a6daa --organization "Robot's Org" --location myLoc
 
-# stream error level logs from a machine part
+# stream logs from a machine
 viam machines part logs --machine 82c608a-1be9-46a5-968d-bad3a8a6daa \
---organization "Robot's Org" --location myLoc --part "myrover-main" --tail true
+--organization "Robot's Org" --location myLoc
+
+# stream logs from a machine part
+viam machines part logs --machine 82c608a-1be9-46a5-968d-bad3a8a6daa \
```
Comment (npentrel) [inline docs/cli.md:1118]: is this a real machine value?

File: docs/cli.md [hunk 7]
```diff
@@ -1101,20 +1104,23 @@ viam machines part restart --machine=<machine id> --part=<part id>
...
+--organization "Robot's Org" --location myLoc --part myrover-main --tail true
```
Comment (npentrel) [inline docs/cli.md:1119]: the part must be specified as an id as far as I can tell testing this.

File: docs/cli.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/cli.md:1059]: And please go through the other commands and actually test them while you have internet

File: docs/cli.md (hunk 4 above)
Comment (sguequierre) [reply docs/cli.md:1333]: Yes, sorry I did not realize this was in scope. Will do

File: docs/cli.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/cli.md:1123]: Sounds good

File: docs/cli.md (hunk 6 above)
Comment (sguequierre) [reply docs/cli.md:1118]: Not sure but it looks like it so I will change to 123

File: docs/cli.md (hunk 7 above)
Comment (sguequierre) [reply docs/cli.md:1119]: I found this to work: 
``` viam machines part run --organization=84318326-eb77-450a-88ec-32a043b793a8 --location=0b2qnylnp0 --machine=collect-data --part=collect-data-main --data '{"name": "vision-1", "camera_name": "camera-2", "detector_name": "mlmodel-1", "n":1}' \
viam.service.vision.v1.VisionService.GetDetectionsFromCamera
```
so I don't think it actually needs to be specd to id!

File: docs/cli.md [hunk 8]
```diff
@@ -1218,39 +1291,59 @@ For more information, see [Train a Model with a Custom Python Training Script](/
...
+```sh {class="command-line" data-prompt="$"}
+# submit training job on data in Viam cloud with a Viam-managed training script
+viam train submit managed --dataset-id=456 --model-org-id=123 --model-name=MyCoolClassifier --model-type=single_label_classification --model-labels=1,2,3
+
+# submit custom training job with an existing training script in the Registry on data in Viam cloud
+viam train submit custom from-registry --dataset-id=<INSERT DATASET ID> --org-id=<INSERT ORG ID> --model-name="MyRegistryModel" --model-version="2" --version="1" --script-name="mycompany:MyCustomTrainingScript"
+
+# submit custom training job with an uploaded training script on data in Viam cloud
+viam train submit custom with-upload --dataset-id=<INSERT DATASET ID> --model-org-id=<INSERT ORG ID> --model-name="MyRegistryModel" --model-type="single_label_classification" --model-version="2" --version="1" --path=<path-to-tar.gz> --script-name="mycompany:MyCustomTrainingScript"
+```
```
Comment (npentrel) [inline docs/cli.md:1323]: are the quotes needed with the equal sign?

File: docs/cli.md [hunk 9]
```diff
@@ -1218,39 +1291,59 @@ For more information, see [Train a Model with a Custom Python Training Script](/
...
+
 #### Command options
 
 <!-- prettier-ignore -->
 | Command option | Description | Positional arguments |
 | -------------- | ----------- | -------------------- |
-| `submit` | Submits training job on data in the Viam cloud | `managed`, `custom` |
+| `submit` | Submits training job on data in the Viam cloud. | `managed`, `custom` |
```
Comment (npentrel) [inline docs/cli.md:1330]: get, cancel, list are also options

File: docs/cli.md (hunk 8 above)
Comment (sguequierre) [reply docs/cli.md:1323]: not needed, don't make invalid either--will remove

File: docs/cli.md (hunk 9 above)
Comment (sguequierre) [reply docs/cli.md:1342]: added! sorry I just didn''t get to that yet

File: docs/cli.md (hunk 4 above)
Comment (sguequierre) [reply docs/cli.md]: done

File: docs/cli.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/cli.md]: ok have tested fully all now except for module upload!

File: docs/cli.md [hunk 10]
```diff
@@ -523,25 +528,25 @@ done
 | -------- | ----------- | ------------------- | --------- |
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/cli.md:528]: ahh i didn't even realize these were options that we were supposed to document bc you can only get them from the filter i think? but i will try and add

[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/cli.md:528]: added

**Review by npentrel — APPROVED** [review]

**Comment by npentrel** [comment]
Thanks - let's get this one merged. I have some more changes in a separate PR that I'll rebase onto your changes once merged

**Pull request merged**